.pre-commit-config.yaml

```diff
@@ -0,0 +1,29 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-json
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: fix-encoding-pragma
+      - id: no-commit-to-branch
+      - id: pretty-format-json
+        exclude: Pipfile.lock
+      - id: trailing-whitespace
+  - repo: https://github.com/psf/black
+    rev: 19.3b0
+    hooks:
+      - id: black
+  - repo: https://github.com/asottile/reorder_python_imports
+    rev: v2.3.5
+    hooks:
+      - id: reorder-python-imports
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: '3.8.3'
+    hooks:
+      - id: flake8
+  - repo: https://github.com/hukkinj1/mdformat
```

Comment:
this formatter seems to break the current readme, so maybe we don't include it for now. 
but I like the idea of having enforced md standards as well.

Reply:
the current readme was out of format so it was breaking. I had to manually fix it. from now on it would work.